apps/api/src/hyp3_api/api-spec/openapi-spec.yml

```diff
@@ -154,7 +154,7 @@ components:
       description: Contains a list of new job objects.
       type: array
       minItems: 1
-      maxItems: 200
+      maxItems: 1000
       items:
```

Comment:
There are valid and unanswered questions about doing this, I think it would be okay to send it to test, but without knowing the extent to which this will effect latency I'm hesitant to send this change to production